protoc_plugin/lib/src/gen/google/api/client.pb.dart

```diff
@@ -53,7 +53,7 @@ class CommonLanguageSettings extends $pb.GeneratedMessage {
       createEmptyInstance: create)
     ..aOS(1, _omitFieldNames ? '' : 'referenceDocsUri')
     ..pc<ClientLibraryDestination>(
-        2, _omitFieldNames ? '' : 'destinations', $pb.PbFieldType.KE,
+        2, _omitFieldNames ? '' : 'destinations', $pb.PbFieldTypeInternal.KE,
```

Comment:
My read on the `PbFieldType` => `PbFieldTypeInternal` change is that the generated code now needs to depend on a new (major?) version of `package:protobuf`. It may be best to indicate that via a major version bump for `protoc_plugin`, so that users have to opt-into the new generator.